tests/tdastro/test_graph_state.py

```diff
@@ -387,11 +386,11 @@ def test_graph_to_from_file():
     state.set("b", "v1", [6.0, 7.0, 8.0])
 
     with tempfile.TemporaryDirectory() as dir_name:
-        file_path = os.path.join(dir_name, "state.ecsv")
-        assert not Path(file_path).is_file()
+        file_path = Path(dir_name, "state.ecsv")
```

Comment:
This one is a slightly larger change recommendation, but:

pytest provides a standard fixture `tmp_path`, which gives you a temp path per test. And it's already a `Path` instance!

```
def test_graph_to_from_file(tmp_path):
  ...
  file_path = tmp_path / "state.ecsv"
  ...
```

Reply:
Done